demos/android/MASVS-STORAGE/MASTG-DEMO-0068/MASTG-DEMO-0068.md

```diff
@@ -1,9 +1,33 @@
 ---
 platform: android
-title: Reserved for MASWE-0006 - Unencrypted SQLite Storage (Alternative Demo)
+title: Sensitive Data in Unencrypted SQLite
 id: MASTG-DEMO-0068
 code: [kotlin]
 test: MASTG-TEST-0304
-status: placeholder
-note: This placeholder reserves the ID for a potential alternative or expanded demo related to unencrypted SQLite storage, linked to the MASTG-TEST-0304 test case.
+status: new
 ---
+
+### Sample
+
+The snippet below shows sample code that uses the default SQLite API (`context.openOrCreateDatabase`) to store sensitive data, including PII and an access token, without any encryption.
+
+{{ MastgTest.kt }}
+
+### Steps
+
+1. Install the application on a test device.
+2. Open the app and exercise it to trigger the database creation and data insertion.
+3. Execute the analysis script **`run.sh`**.
+4. The script pulls the database file (`PrivateUnencryptedData.db`) and queries its content.
+
+{{ run.sh }}
+
+### Observation
+
+The `output.txt` file contains the extracted content from the `Users` table, showing the sensitive PII (email address) and the access token stored in **plaintext**.
+
+{{ output.txt }}
+
+### Evaluation
+
+This test fails because the application stores highly sensitive data (Access Token and PII like the user's email address) using the standard SQLite API. By default, SQLite stores data in an unencrypted file within the app's private sandbox, violating the **MASVS-STORAGE** requirement (MASWE-0006). This data is trivially accessible if an attacker achieves privileged access to the device (e.g., via root).
```

Comment:
It isn't wrong what you added, but I removed part of it to keep it consistent with other DEMOs.

```suggestion
This test fails because the app uses the standard SQLite API to store sensitive data, specifically an access token (secret) and the user's email address (PII), in its sandbox without additional encryption.
```